packages/synapse-react-client/src/components/TermsAndConditions/TermsAndConditions.tsx

```diff
@@ -123,21 +81,31 @@ const TermsAndConditions: React.FunctionComponent<TermsAndConditionsProps> = ({
 
   // Placeholder function to check if all checkboxes are checked and agreement is signed
   const checkFormCompleted = () => {
-    const allCheckboxChecked = !checkboxChecked.includes(false)
+    const allCheckboxChecked =
+      checkboxChecked.length > 0 && !checkboxChecked.includes(false)
```

Comment:
When calling back, all checkboxes have been selected only if there is at least one checkbox being shown